### DISCUSSION TITLE
Z offset flsun v400

### DISCUSSION THREAD

**Discussion by Duiling01** [Guilouz/Klipper-Flsun-Speeder-Pad#146] [General]
Hello, impossible to do Z OFFSET on my flsun v400.
When I do a PROBE_CALIBRATE with the probe and I want to validate I get an error message 'Failled adjust first position'
When I want to do my Z offset with the probe I have too much distance from the plate and when I do the Z offset without the probe, my nozzle hits the bed. I have already broken 1 probe and 4 nozzles

Sorry i use translator, speak only french

**Comment by kaktuss77** [comment]
hello,
When you use this macro, in first time you need to have the probe. after the head has finish the 3 probes (when the position is low but not at home place), you have to take off the probe from the head and then do Z offset with the buttons and a piece of paper 

it's like that I have do mine.

(sorry for my poor english)

**Comment by Duiling01** [comment]
Good morning

I thank you

That's what I'm doing, when I do it this way when I start my print the nozzle is much too high. When I want to do it without the probe my nozzle hits the bed.
I reinstalled the official version 1.4 of the speeder pad and I have no more problems. I only have the problem with the Guilouz configuration

Kind regards

**Reply by SckyzO** [reply]
Ho ! So just read the doc :) 
You need to start a Z offset, and save ... just save ... After you can do a bed leveling ...

**Reply by SckyzO** [reply]
I can tell you, I do the same error like you !! 1 hour to try to understand ... just save And it's work
